tutorials/llm/llama-3/pruning-distillation/03_a_depth_pruning.ipynb

```diff
@@ -0,0 +1,77 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "8bc99d2f-9ac6-40c2-b072-12b6cb7b9aca",
+   "metadata": {},
+   "source": [
+    "### Step 3: Prune the finetuned-teacher model to create a student\n",
+    "In this step, we will explore two methods to prune the finetuned teacher model. Refer to the ``NOTE`` in the **_step-by-step instructions_** section of [introduction.ipynb](./introduction.ipynb) to decide which pruning techniques you would like to explore.\n",
+    "\n",
+    "In the first method, depth-pruning, we trim the layers of the model."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "72fa494e-6268-4044-a1d6-c0518d450cfd",
+   "metadata": {},
+   "source": [
+    "#### Step 3.a.: Using depth-pruning \n",
+    "To depth-prune, we will trim the last 16 layers in the finetined teacher model. For depth-pruning, we would be using the [megatron_gpt_drop_layers](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/language_modeling/megatron_gpt_drop_layers.py) script. \n",
+    "\n",
+    "Per the [blog](https://developer.nvidia.com/blog/how-to-prune-and-distill-llama-3-1-8b-to-an-nvidia-llama-3-1-minitron-4b-model/) and [tech report](https://arxiv.org/pdf/2408.11796), removing contiguous layers from the second last block (layers 16 to 31 continuously) yields the best overall results. \n",
+    "\n",
+    "> `NOTE:` In the block of code below, pass the paths to your finetuned teacher .nemo model."
```

Comment:
fix punctuation

`NOTE:` In the block of code below, pass the paths to your fine-tuned teacher .nemo model."